Azure/azure-cli
# [APIM] Add Named Value operations

**Description**  

This PR contains code that adds Named Value operations to the az apim command module. The following operations were added:

- apim nv create
- apim nv update
- apim nv delete
- apim nv show
- apim nv list

**Testing Guide**  
I have extended the testing scenario contained in the apim command module folder. For testing execute `azdev test apim` in the virtual environment. The testing sequence I implemented is described here:
1. First I create a named value
2. Then I get the named value
3. After these two operations i update one Named Value
4. The fourth step contains listing of all named values (which should equal a count of one)
5. And at the end I delete that Named Value.

**History Notes**  
[APIM] Added general az apim named value commands

---

This checklist is used to make sure that common guidelines for a pull request are followed.
